cupy/cuda/nvtx.pyx

```diff
@@ -0,0 +1,84 @@
+"""
+Wrapper for NVIDIA Tools Extension Library (NVTX)
+
+"""
+cimport cython
+
+
+cdef int num_colors = 10
+cdef uint32_t colors[10]
+colors[0] = 0x00FF00
+colors[1] = 0x00FF7F
+colors[2] = 0x007FFF
+colors[3] = 0x0000FF
+colors[4] = 0x7F00FF
+colors[5] = 0xFF007F
+colors[6] = 0xFF0000
+colors[7] = 0xFF7F00
+colors[8] = 0x7FFF00
+colors[9] = 0x7F7F7F
+
+
+cpdef void Mark(str message, int id_color=-1) except *:
```

Comment:
How about splitting the function?
One function receive a color code, other function receive a color id.